matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/SessionModule.kt

```diff
@@ -387,4 +389,10 @@ internal abstract class SessionModule {
 
     @Binds
     abstract fun bindEventSenderProcessor(processor: EventSenderProcessorCoroutine): EventSenderProcessor
+
+    @Binds
+    abstract fun bindToDeviceService(deviceService: DefaultToDeviceService): ToDeviceService
+
+    @Binds
+    abstract fun bindEventStreamService(deviceService: DefaultEventStreamService): EventStreamService
```

Comment:
Could be grouped with the other services. Also rename param to `service` (at 2 places)

Reply:
done